test/index.test-d.ts

```diff
@@ -45,6 +46,7 @@ const runServer = async () => {
     }, (err) => {
       throw err
     }, {
+      headers: { 'content-type': 'multipart/form-data' },
```

Comment:
Just curious why it is added?

Reply:
it was always mandatory, updated types reflect that

Reply:
the content-type-header is mandatory in the typings